lisa/node.py

```diff
@@ -161,6 +175,25 @@ def is_linux(self) -> bool:
         self.initialize()
         return self.os.is_linux
 
+    @property
+    def support_sudo(self) -> bool:
+        self.initialize()
+
+        # check if sudo supported
+        if self.is_linux and self._support_sudo is None:
+            process = self._execute("command -v sudo", shell=True, no_info_log=True)
```

Comment:
can we use like sudo -V, then check whether it exist or not?

Reply:
what can `sudo -V` help? Is there any compatibility issue we found on different sudo version?

Reply:
in case, command 'command' doesn't exist, sudo -V is to display the version.

Reply:
`command` is heavily used to detect every "command". If it's not here, no case can pass. So it doesn't need to use a different way to `sudo` only.